src/udev/udev-builtin-usb_id.c

```diff
@@ -75,11 +76,9 @@ static void set_usb_iftype(char *to, int if_class_num, size_t len) {
 
 static int set_usb_mass_storage_ifsubtype(char *to, const char *from, size_t len) {
         int type_num = 0;
-        char *eptr;
         const char *type = "generic";
 
-        type_num = strtoul(from, &eptr, 0);
-        if (eptr != from) {
+        if (safe_atoi(from, &type_num) >= 0) {
```

Comment:
Not a matter, but `safe_atou()`?

Reply:
We're parsing into an int here. So `safe_atoi` is right.